### PR TITLE
Update free-courses-hi.md

### PR DESCRIPTION
**Pull request by biswabaibhab007**
## What does this PR do?
- Added Express.js under Javascript
- Added a video course for Express,js

### Description
- The tutor explains various use cases of express.js like user authentication

### Why is this valuable (or not)?
- Earlier there were no resources for express.js in this repo

### How do we know it's really free?
- It's on youtube

### For book lists, is it a book? For course lists, is it a course? etc.
- It's a course

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Ajay-singh1** [comment]
Go for it buddy fix the linter errors.

**Comment by biswabaibhab007** [comment]
> Go for it buddy fix the linter errors.

done 👍

**Review by Ajay-singh1 — APPROVED** [review]
LGTM!

**Review by sohansai — COMMENTED** [review]
We prefer 'playlist' links for YouTube contributions. Avoid shortened links, and keep the resource's name unchanged when contributing.

**Comment by biswabaibhab007** [comment]
> We prefer 'playlist' links for YouTube contributions. Avoid shortened links, and keep the resource's name unchanged when contributing.

@sohansai But express.js is not a big topic and this video covers everything that is actually used in real life nothing extra.

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -402,6 +403,11 @@
 * [Web Development Course](https://www.youtube.com/playlist?list=PLfqMhTWNBTe3H6c9OGXb5_6wcc1Mca52n) - Apna College
 
 
+### Express.js
+
+* [Express.js - Learn What Matters](https://youtu.be/pKJ4GGyDgJo?si=YUJf86FH54wofl99) - Sheryians Coding School
```
Comment (eshellman) [inline courses/free-courses-hi.md:408]: we don't accept shortened links like this one. single videos are only acceptable if there is no playlist and the lengthe is longer than one hour, as this one is.
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. For this one, you can just add it to the javascript category

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -402,6 +403,11 @@
 * [Web Development Course](https://www.youtube.com/playlist?list=PLfqMhTWNBTe3H6c9OGXb5_6wcc1Mca52n) - Apna College
 
 
+### Express.js
```
Comment (eshellman) [inline courses/free-courses-hi.md:406]:
```suggestion
#### Express.js
```
subcategory

**Review by Ajay-singh1 — APPROVED** [review]
This Looks Good By the way.

**Review by Ajay-singh1 — CHANGES_REQUESTED** [review]
The URl is a shortened one fix that first than add the list to the javascript section.

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -390,6 +390,7 @@
 ### JavaScript
 
 * [Chai aur Javascript हिन्दी](https://www.youtube.com/playlist?list=PLu71SKxNbfoBuX3f4EOACle2y-tRC5Q37) - Chai aur Code (Hitesh Choudhary)
+* [Express.js - Learn What Matters](https://m.youtube.com/watch?v=pKJ4GGyDgJo&pp=ygUUZXhwcmVzcyBqcyBzaGVyeWlhbnM%3D) - Sheryians Coding School
```
Comment (eshellman) [inline courses/free-courses-hi.md:393]:
```suggestion
* [Express.js - Learn What Matters](https://youtube.com/watch?v=pKJ4GGyDgJo&pp=ygUUZXhwcmVzcyBqcyBzaGVyeWlhbnM%3D) - Sheryians Coding School
```

File: courses/free-courses-hi.md [hunk 4]
```diff
@@ -390,6 +390,7 @@
 ### JavaScript
 
 * [Chai aur Javascript हिन्दी](https://www.youtube.com/playlist?list=PLu71SKxNbfoBuX3f4EOACle2y-tRC5Q37) - Chai aur Code (Hitesh Choudhary)
+* [Express.js - Learn What Matters](https://www.youtube.com/watch?v=pKJ4GGyDgJo&pp=ygUUZXhwcmVzcyBqcyBzaGVyeWlhbnM%3D) - Sheryians Coding School
```
Comment (sohansai) [inline courses/free-courses-hi.md:393]:
```suggestion
* [Express.js - Learn What Matters](https://www.youtube.com/watch?v=pKJ4GGyDgJo) - Sheryians Coding School
```

**Comment by biswabaibhab007** [comment]
Done 👍

**Comment by eshellman** [comment]
Done! 
@Ajay-singh1 thanks for assistance.

**Pull request merged**